README.md

```diff
@@ -140,6 +140,11 @@ In addition to `fastlane`'s commands, you also have access to these `fastlane` t
 - [`scan`](https://github.com/fastlane/fastlane/tree/master/scan): The easiest way to run tests for your iOS and Mac apps
 - [`precheck`](https://github.com/fastlane/fastlane/tree/master/precheck): Check your app using a community driven set of App Store review rules to avoid being rejected
 
+## Where can fastlane run?
+Currently _fastlane_ is officially supported on macOS. Support for other operating systems is limited and untested at this point in time. _fastlane_ uses system apis that may not be implemented on other platforms (one example: we make use of `fork()` and it is not supported on Windows). 
+
+**Note:** If you'd like to add support for your favorite platform, feel free to [contribute a PR](https://github.com/fastlane/fastlane/tree/master/YourFirstPR.md)!
```

Comment:
A little nervous about offering to entertain this PR but it would be nice to support Windows and I think in the long run we should.